content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md

```diff
@@ -0,0 +1,157 @@
+---
+weight: 40
+title: Release 10.16.0.0
+layout: bundle
+---
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>DTM application is integrated with Gainsight to capture the data.</td>
+<td>CTM-130</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Graceful handling of custom property and asset type deletion from backend is implemented.</td>
+<td>CTM-141</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>On delete operation in Property library page confirmation pop-up is shown.</td>
+<td>CTM-159</td>
+</tr>
+
+<tr>
+<td>Digital Twin Manager</td>
+<td>Implemeted maximum file size field for custom property of file type.</td>
```

Comment:
```suggestion
<td>Implemented a maximum file size field for custom properties of type file.</td>
```
please check if this is the meaning that was intended.

Reply:
```suggestion
<td>Implemeted maximum file size field for the custom property "file type".</td>
```